.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+name: CI
+
+on: [push, pull_request]
```

To avoid duplicate builds, I recommend this instead:
```suggestion
on:
  push:
    branches: [master]
  pull_request:
```